Gems/Atom/Feature/Common/Assets/Passes/DiffuseGlobalIllumination.pass

```diff
@@ -26,6 +26,54 @@
                     "Name": "DepthStencilInputOutput",
                     "SlotType": "InputOutput",
                     "ScopeAttachmentUsage": "DepthStencil"
+                },
+                {
+                    "Name": "IrradianceOutput",
+                    "SlotType": "InputOutput",
+                    "ScopeAttachmentUsage": "RenderTarget",
+                    "LoadStoreAction": {
```

Comment:
This needs a todo and a comment.

//TODO: Jira[xxxx] - Normally we do not want to do a clear in a parent pass as it generates a slow clear pass which is in-effecient. Investigate how we can remove the clear here and directly attach it when this resource is used.

Reply:
Discussed offline - this will be an implementation change, so the pass files will remain the same.